features/data_converter/json_protobuf/README.md

```diff
@@ -2,14 +2,16 @@
 
 Protobuf values can be converted to and from `json/protobuf` Payloads.
 
-This feature:
+Steps:
 
-- runs a [`BinaryMessage`](../messages.proto) with a single byte `0000 0101` for `data` through the default Payload
-  Converter, writes it to `payloads/json_protobuf.[lang]`, and verifies it matches the other files in `payloads/`
-- decodes all files in `payloads/` with the default Payload Converter and verifies the value is a `Binary Message` with
-  `data: 0000 0101`
+- run a workflow that returns [`DataBlob`](https://pkg.go.dev/go.temporal.io/api/common/v1#DataBlob)
+- verify client result is [`DataBlob`] with value `0xdeadbeef`
+- get result payload of WorkflowExecutionCompleted event from workflow history
+- verify payload encoding is `json/protobuf`, unmarshall its data into a
+`DataBlob` using `jsonpb` library, and compare it to the client
+result
 
 # Detailed spec
 
 `metadata.encoding = toBinary("json/protobuf")`
-`metadata.messageType = toBinary("BinaryMessage")` (used by languages that cannot get a parameter's type at runtime)
\ No newline at end of file
+`metadata.messageType = toBinary("DataBlob")` (used by languages that cannot get a parameter's type at runtime)
```

Comment:
Should be the fully qualified message type.

Reply:
Done